server/firebase.json

```diff
@@ -1 +1,10 @@
-{}
+{
```

Comment:
what is this used for?

Reply:
Agree I'll remove it

Reply:
Agree, I'll remove it

Reply:
Btw, it just contains config for the firebase nodejs environment (like emulators)